src/libraries/Microsoft.PowerFx.Core/App/IExternalEnabledFeatures.cs

```diff
@@ -37,5 +39,7 @@ internal sealed class DefaultEnabledFeatures : IExternalEnabledFeatures
         public bool IsDynamicSchemaEnabled => true;
 
         public bool IsEnhancedComponentFunctionPropertyEnabled => true;
+
+        public bool IsComponentFunctionPropertyDataflowEnabled => true;
```

Comment:
>true [](http://example.com/codeflow?start=66&length=4)

Should this be false?

Reply:
DefaultEnabledFeatures is not used by canvas host so this value here can be treated as dummy value.